.travis.yml

```diff
@@ -11,6 +11,7 @@ script:
     - flake8 .
     - make test-docstrings
     - make docs
+    - cp robottelo.properties.sample robottelo.properties
```

Comment:
As `settings.configure` is being called at testing initialization, in travis the file should be created based in sample file, else we get IOerror